spec/models/user_spec.rb

```diff
@@ -297,4 +297,25 @@
       expect { do_purge }.to change(User, :count).by(0)
     end
   end
+
+  #sam Unit test
+  describe User do
+    let(:emails) {  ['a@a.com','b@b.com'] }
+    let(:users) { double('Users') }
+        it '.superadmin_emails' do 
+          expect(users). to receive(:pluck).with(:email).and_return(emails)
+          expect(User).to receive(:superadmins).and_return(users)
+          expect(User.superadmin_emails).to eq emails
+        end
+  end
+
+  #integration test 
```

Comment:
@decareano if you want to have some placeholder code for the duration of the pull request that's just fine :-)  As far as the integration test goes, having the two factories is fine

I'm not sure what you mean relating to send_email_to_superadmin_about_org_edit - the workflow there should not affect what we are doing in these unit and integration tests